ur_robot_driver/doc/resources/README.md

```diff
@@ -0,0 +1,30 @@
+# Resources about ROS 2 Universal Robot driver
+
+The files in this folder are available for use under [![License](https://i.creativecommons.org/l/by/4.0/88x31.png)](http://creativecommons.org/licenses/by/4.0/) license.
+
+
+## Diagrams and images
+
+- [ros2_control diagrams](ros2_control_ur_driver.drawio) - `.drawio` file, Author: [Denis Stogl](mailto:denis.stogl@picknik.ai) (PickNik Robotics)
+- [generated_images](generated_images/) - see individual files
+
+
+## 2021-10 ROS World presentation
+
+[Presentation: Making a robot ROS 2 powered - a case study using the UR manipulators](2021-10_ROS_World_2021_Making_a_robot_ROS2_powered.pdf)
```

Comment:
Looks like the pdf compressed down fairly nicely to <5mb. @urrsk would you like this added to the repo?

@destogl this could get updated in the future, maybe we should set up git lfs for .pdfs

Reply:
The size is ok